app/controllers/atmosphere/api/v1/appliance_types_controller.rb

```diff
@@ -133,7 +133,14 @@ def filter
         end
 
         def appliance_type_params
-          params.require(:appliance_type).permit(:name, :description, :shared, :scalable, :visible_to, :author_id, :preference_cpu, :preference_memory, :preference_disk, :appliance_id)
+          allowed_params = [
+                             :name, :description, :shared, :scalable,
```

Comment:
Use 2 spaces for indentation in an array, relative to the start of the line where the left bracket is.